src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -32,7 +32,7 @@ private void InitSort(CultureInfo culture)
             }
             else
             {
-                _isAsciiEqualityOrdinal = (_sortName == "en-US" || _sortName == "");
+                _isAsciiEqualityOrdinal = (_sortName == "en-US" || _sortName == "" || _sortName == "en-GB");
```

Comment:
Why do we hardcode a list rather than initializing based on comparing ABCD... to abcd... with ICU with ignore case, ala what we do with TextInfo._isAsciiCasingSameAsInvariant?

Reply:
Is it not sufficient. The culture-sensitive comparison are context sensitive, they cannot be done one character at a time in general.

@adamsitnik showed one of the many examples above: In Polish or Czech, `"ch".StartsWith("c")` needs to return false.

Reply:
I see. I already had this open on my phone and hadn't seen the other discussion when I asked.